functions/pipeline/shared/db_access/db_access_v2.py

```diff
@@ -155,6 +152,98 @@ def get_tag_complete_images(self, number_of_images, user_id):
             conn.close()
         return tag_complete_images
 
+    def get_ready_to_tag_images(self, number_of_images, user_id):
```

Comment:
@samiyaakhtar This is duplicated code with get_tag_complete_images. The only change is the status type. Perhaps we can cut down on the duplication?

Reply:
There's difference in the limit n, which is why I created a separate function for now, but I will merge them.